pkg/cosign/tuf/client.go

```diff
@@ -57,7 +55,7 @@ var rootClient *client.Client
 var rootClientMu = &sync.Mutex{}
 
 func GetEmbeddedRoot() ([]byte, error) {
-	return root.ReadFile(filepath.Join("repository", "root.json"))
+	return root.ReadFile("repository/root.json")
```

Comment:
Why not `filepathJoin`?

Reply:
that'd be an option, but behavior depends on whether or not we're reading from the embedded FS or real FS, so it'd make more sense to create an interface

Reply:
created private global functions, PTAL